index.bs

```diff
@@ -114,16 +115,16 @@ API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Ambient Light Sensor's associated {{Sensor}} subclass
-is the {{AmbientLightSensor}} class.
+The <dfn>Ambient Light Sensor</dfn> <a>sensor type</a>'s associated {{Sensor}}
+subclass is the {{AmbientLightSensor}} class.
 
-The Ambient Light Sensor has a <a>default sensor</a>,
+The <a>Ambient Light Sensor</a> has a <a>default sensor</a>,
 which is the device's main light detector.
 
-The Ambient Light Sensor's <a>reporting mode</a> is <a>implementation
+The <a>Ambient Light Sensor</a>'s <a>reporting mode</a> is <a>implementation
```

Comment:
this paragraph might be dropped, pls see https://github.com/w3c/sensors/pull/277

Reply:
Dropped in https://github.com/w3c/ambient-light/pull/40/commits/a50121b272e37b8e425de6619d264e9c6339c99c